### ISSUE TITLE
Error 500 : widget_ manager enable version 5.1 download via github

### ISSUE THREAD

**Issue by olairmarcos** [ColdTrick/widget_manager#71]
Hello, When you enable version 5.1 download via github appears error 500. The download version via Elgg site works normal. My elgg version 1.12.4, Arvixe company server.
Thanks for help. Wait.

**Comment by jdalsem** [comment]
Did you install it with composer? There are dependencies that need to be installed. Composer will take care of that.

**Comment by olairmarcos** [comment]
The error 500 is related to the line 6 start.php file. I see that in the .zip file widget_manager there is no vendor folder. So I commented the line 6 in start.php File widget_manager and works properly. Am I doing this right?
I should install the new version of widget_manager over the existing version on the site?
Generally I download the .zip file from GitHub and decompress the site of the mod folder as a new widget. I never write on the old widget.
What is the correct way to do?
Thank you for your help.

**Comment by jdalsem** [comment]
I see now that we did not post the correct download on the releases tab https://github.com/ColdTrick/widget_manager/releases. It is missing the dependency. That is the only difference between 5.1 on the Elgg community and here on github. If you are downloading directly from the code repository you still need to use composer (see more info here: https://elgg.org/discussion/view/2262506/composer-welgg-and-plugins).

I have updated the release here on github to include the vendor folder. Thanks for reporting.
